lib/helpers/pack_hoodie.js

```diff
@@ -105,20 +106,21 @@ module.exports = function (env_config) {
   // Create the `bundleConcat` that we'll use to create the `browserify` bundle.
   fs.writeFileSync(bundleConcat, bundleTmpl + concatPlugins(plugins));
 
-  var b = browserify(bundleConcat);
-
-  return b.bundle({
+  var b = browserify(bundleConcat, {
     standalone: 'Hoodie',
     external: 'jquery'
-  }, function (err, bundle) {
+  });
+
+  b.bundle(function (err, bundle) {
     if (err) { return console.error(err); }
     // Store successful bundle in cache.
     cache = bundle;
     // Clean up! Remove dynamically generated file with concatenated scripts
     // once we are done building the bundle.
-    fs.unlink(bundleConcat, function (err) {
+    fs.unlinkSync(bundleConcat, function (err) {
```

Comment:
I think I'd either return a value if the method is synchronous and try/catch eventual errors, or I'd make the callback follow done conventions and have it pass error as first argument. `fs.unlinkSync` could trough an error, in theory.